pkg/model/account.go

```diff
@@ -0,0 +1,49 @@
+package model
+
+// Account - Handles importing and exporting Accounts and non-exported Properties
+type Account struct {
+	ID                  *string                `json:"id,omitempty" dynamodbav:"Id" schema:"id"`                                                                // AWS Account ID
```

Comment:
What's the purpose schema tag? Please clarify.

Reply:
I found answer to my question. Package gorilla/schema fills a struct with form values based on schema tag.

Reply:
https://stackoverflow.com/questions/50536368/does-gorilla-schema-support-the-json-reflection-tag-natively

It looks like we can use the same json tag too?

Reply:
Removing the schema for now.  Good find on using the json tag.  I'll try it out when that part gets implemented or add back the scheme.  

Some of this is leftover of breaking out the large PR into smaller ones.  Not everything is getting used in this state but the next PR will use it.